workers/External/spatialos_worker_packages.json

```diff
@@ -1,11 +1,20 @@
 {
   "targets": [
+    {
+      "path": "../../dependencies/worker_sdk",
+      "type": "worker_sdk",
+      "packages": [
+        {
+          "name": "cpp_headers"
+        }
+      ]
+    },
     {
       "path": "../../dependencies/worker_sdk/win32/debug",
```

Comment:
I suggest extracting to `lib/win32/debug`, `lib/win32/release`, `lib/macos` and `lib/linux` to maintain the old directory structure.

Reply:
Changed to `win32/debug/lib` etc. to match the previous directory structure.